lib/libglvnd/BUILD

```diff
@@ -0,0 +1,6 @@
+if module_installed mesa-lib; then
```

Comment:
This is just wrong, please revert all libglvnd changes. It's working as intended.